infra/spawn_data_pipeline.py

```diff
@@ -79,7 +79,7 @@ def main():
 
     for task in raw_tasks["tasks"]:
         # Try render the task template
-        context = {}
+        context = {"version": docker_tag}
```

Comment:
Nit: since you use it only here, you might as well do `"version": os.getenv("TAG", "latest")`

Reply:
Perhaps we could even remove TAG from the env and use get_bugbug_version you just introduced, but that's not a big difference